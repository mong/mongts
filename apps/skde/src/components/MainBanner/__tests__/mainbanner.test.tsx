/**
 * To view a rendered version of all the snapshots:
 *
 * Copy the contents of ./__snapshots__/index.tsx.snap
 * Go to https://codesandbox.io/s/snapshot-viewer-lqjjsm?file=/src/snapshots.js
 * Replace the contents with what you copied
 */

import { render } from "@testing-library/react";
import { MainBanner } from "..";

test("BokmÃ¥l render", async () => {
  const { container } = render(<MainBanner lang="nb" />);
  expect(container).toMatchSnapshot();
});

test("English render", async () => {
  const { container } = render(<MainBanner lang="en" />);
  expect(container).toMatchSnapshot();
});
