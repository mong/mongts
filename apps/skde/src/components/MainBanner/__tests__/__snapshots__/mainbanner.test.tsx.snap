// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bokmål render 1`] = `
<div>
  <div
    class="main"
  >
    <div
      class="headerMain"
    >
      <div
        class="pageImage"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%271184%27%20height=%27435%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="mainpage photo"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <h1
          class="pageTitle"
        >
          Likeverdige helsetjenester – uansett hvor du bor?
        </h1>
      </div>
    </div>
    <div
      class="pageTitle"
    />
    <div
      class="pageIngressContent"
    >
      <div
        class="pageIngress"
      >
        Helseatlas sammenlikner befolkningens bruk av helsetjenester i forskjellige geografiske områder, uavhengig av hvilket sted pasientene behandles.
      </div>
    </div>
  </div>
</div>
`;

exports[`English render 1`] = `
<div>
  <div
    class="main"
  >
    <div
      class="headerMain"
    >
      <div
        class="pageImage"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%271184%27%20height=%27435%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="mainpage photo"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
        <h1
          class="pageTitle"
        >
          Equitable health services – regardless of where you live?
        </h1>
      </div>
    </div>
    <div
      class="pageTitle"
    />
    <div
      class="pageIngressContent"
    >
      <div
        class="pageIngress"
      >
        In Norway, it is a goal for the entire population to have an equal supply of health services across geography and social groups. The health atlas is a tool for comparing the population's use of health services in different geographical areas, regardless of where the patients are treated.
      </div>
    </div>
  </div>
</div>
`;
